app/src/main/java/com/example/groxweatherapp/model/WeatherModel.java

```diff
@@ -1,20 +1,20 @@
 package com.example.groxweatherapp.model;
 
+import com.example.groxweatherapp.api.WeatherApiClient;
 import com.example.groxweatherapp.api.WeatherApiClient.RequestMode;
 import com.google.auto.value.AutoValue;
 import java.util.ArrayList;
 import java.util.List;
 
-import static com.example.groxweatherapp.api.WeatherApiClient.INVALID;
-import static com.example.groxweatherapp.model.WeatherModel.WeatherModelState.INITIATED;
+import static com.example.groxweatherapp.model.WeatherModel.WeatherModelState.REFRESHING;
 
 @AutoValue
 public abstract class WeatherModel {
 
     public static WeatherModel INITIAL_STATE = WeatherModel.builder().build();
 
     public enum WeatherModelState {
-        INITIATED, REFRESHING, SUCCESS, ERROR
```

Comment:
Should we change REFRESHING to REFRESH because now the horizontal selector is telling the main presenter to refresh the data?